internal/conf/yaml_config_ops.go

```diff
@@ -230,12 +230,11 @@ func (c *ConfigKeys) AddConfKey(confKey string, reqField map[string]interface{})
 	c.lock.Lock()
 	defer c.lock.Unlock()
 
+	c.dataCfg[confKey] = reqField
 	if _, ok := c.etcCfg[confKey]; ok {
-		return fmt.Errorf("duplicate key %s, already exist in etc folder", confKey)
+		delete(c.etcCfg, confKey)
```

Comment:
Assume dataCfg have higher priority to override etcCfg. Is it better to leave it, since the etc file is not changed?

Reply:
etcCfg just load from disk and save in memory, this delete operation just happen in memory and do not save to disk

main use case is user edit the config key in etc folder and save it. we save the configs in dataCfg and save to disk.